src/output/help_template.rs

```diff
@@ -443,7 +443,10 @@ impl<'cmd, 'writer> HelpTemplate<'cmd, 'writer> {
         }
         ord_v.sort_by(|a, b| a.0.cmp(&b.0));
 
-        let next_line_help = self.will_args_wrap(args, longest);
+        let next_line_help = args
+            .iter()
+            .filter(|arg| self.should_show_arg(arg))
+            .any(|arg| self.arg_next_line_help(arg, longest));
```

Comment:
My problem with this change is the old code made it clearer what the intended case was for wrapping.